CRM/Profile/Form.php

```diff
@@ -500,7 +500,7 @@ public function setDefaultsValues() {
             ) {
               $entityId = $this->_activityId;
             }
-            $url = CRM_Core_BAO_CustomField::getFileURL($entityId, $key);
+            $url = CRM_Core_BAO_CustomField::getFileURL($entityId, $key, $value);
```

Comment:
@lcdservices Can you please update this PR so that we can merge and close the ticket?